site/content/blogs/release-v117.en.md

```diff
@@ -0,0 +1,48 @@
+# AWS Copilot v1.17: Tracing for Request-Driven Web Service.
+
+Copilot v1.17 brings with it a brand-new features and several improvements:
+
+* **Tracing for Request-Driven Web Service:** With the release of AWS X-Ray tracing support for AWS App runner services, you can now add `observability.tracing: awsxray` in your Request-Driven Web Service's 
+  manifest to send your traces to AWS XRay. [See detailed section](./#send-your-request-driven-web-services-traces-to-aws-xray).
+* **Increased visibility for progress trackers:** [See detailed section](./#define-messages-filter-policy-in-publishsubscribe-service-architecture).
+* **Bug fix:** Remove color formatting of suggested pipeline names ([#3437](https://github.com/aws/copilot-cli/pull/3437)).
+
+## What’s AWS Copilot?
+
+The AWS Copilot CLI is a tool for developers to build, release, and operate production ready containerized applications on AWS.  
+From getting started, pushing to staging, and releasing to production, Copilot can help manage the entire lifecycle of your application development.
+At the foundation of Copilot is AWS CloudFormation, which enables you to provision infrastructure as code in a single operation.
+Copilot provides pre-defined CloudFormation templates and user-friendly workflows for different types of micro services creation and operation,
+enabling you to focus on developing your application, instead of writing deployment scripts.
+
+See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
+
+## Send Your Request-Driven Web Service's Traces to AWS X-Ray
+_Contributed by [Wanxian Yang](https://github.com/Lou1415926/)_
+
+You can now send traces generated your Request-Driven Web Services to AWS X-Ray. This allows you to visualize the service map and traces 
+through CloudWatch console or AWS X-Ray console.
+
+To use this feature, your service needs to be first instrumented with 
```

Comment:
do we want to mention [AWS X-Ray SDKs](https://docs.aws.amazon.com/xray/latest/devguide/xray-instrumenting-your-app.html#xray-instrumenting-xray-sdk) as an option?

Reply:
RDWS's tracing works only with ADOT atm!

Reply:
sad :/ sounds good!